src/com/example/pathom.cljs

```diff
@@ -33,15 +34,38 @@
 (pc/defresolver user-authenticated? [env {:keys [current-user]}]
   {::pc/input #{:current-user}
    ::pc/output [:user/authenticated?]}
-  (prn current-user (-> current-user
-                        :user/id
-                        nil?
-                        not))
   {:user/authenticated? (-> current-user
                             :user/id
                             nil?
                             not)})
 
+(def readiness-id "375623e0-92dd-4815-b507-4d577a55f37c")
+
+(defn quiz-by-id* [id]
+  (first (filter #(= id (:quiz/id %)) quizzes)))
+
+#_(defn parse-quiz [id]
+    (let [quiz (first (filter #(= id (get % :id)) quizzes))
+          questions (:questions quiz)]
+      {:quiz/id id :quiz/version (:version quiz)
+       :questions (map (fn [q]
+                         {:question/id (:id q)
+                          :question/label (:label q)
+                          :question/options (map (fn [o]
+                                                   {:option/label (:label o)
+                                                    :option/value (:value o)}) (:options q))}) questions)}))
+
+(pc/defresolver quiz-by-id [_ {id :quiz/id}]
+  {::pc/input #{:quiz/id}
+   ::pc/output [:quiz/id :quiz/version :quiz/label
+                {:quiz/questions [:question/id :question/label
+                                  {:question/options [:option/label :option/value]}]}]}
+  (quiz-by-id* id))
+
+(pc/defresolver readiness-quiz [_ _]
+  {::pc/output [{:readiness-quiz [:quiz/id]}]}
+  {:readiness-quiz {:quiz/id "375623e0-92dd-4815-b507-4d577a55f37c"}})
```

Comment:
this resolver just needs to send back an ident of `[:quiz/id id-value]` for pathom to connect it to the `quiz-by-id` resolver.

This enables queries for the quiz with this specific id without having to send the id in.

This is what the query looks like

```clojure
[{:readiness-quiz
  [:quiz/version
   :quiz/id
   :quiz/label
   {:quiz/questions
    [:question/id
     :question/label
     {:question/options [:option/label :option/value]}]}]}
 :com.wsscode.pathom.core/errors]
```